doc/DreamOS-cygwin.md

```diff
@@ -0,0 +1,327 @@
+If you have 2 cores, and so on.
+
+### Binutils
+	
+   	cd $HOME/src
+    mkdir build-binutils
+    cd build-binutils
+    ../binutils-x.y.z/configure --target=$TARGET --prefix="$PREFIX" --with-sysroot --disable-nls --disable-werror
+    make
+    make install
+
+### GCC
+
+First we need to download essential source code needed for GCC, if you haven't got it in our environment.
+
+	cd gcc-x.y.z
+    contrib/download_prerequisites
+After download finishes start to build
+
+    cd $HOME/src
+    which -- $TARGET-as || echo $TARGET-as is not in the PATH
+    
+    mkdir build-gcc
+    cd build-gcc
+    ../gcc-x.y.z/configure --target=$TARGET --prefix="$PREFIX" --disable-nls --enable-languages=c,c++ --without-headers
+    make all-gcc
+    make all-target-libgcc
+    make install-gcc
+    make install-target-libgcc
+
+### Testing the cross compiler
+
+If everything worked well, typing
+
+	i686-elf-gcc --version
+    
+will show the version of your cross compiler :)
+
+**IMPORTANT: append C:\\"cygwin path"\opt\cross\bin\ to your Windows PATH otherwise the cross compiler will be not available restarting terminal.**
+    
+## Software for DreamOS build
+
+To build DreamOS we need to download and install some software for Windows:
+
+* [Bochs](https://sourceforge.net/projects/bochs/) (emulator)
+* [Ext2Fsd](https://sourceforge.net/projects/ext2fsd/files/) (ext2/ext3 filesystem drivers for Windows, needs reboot)
+* [Git](https://git-scm.com/download/win) (to get DreamOS repository)
+* [Nasm](http://www.nasm.us/) (assembler)
+* [Osfmount](http://www.osforensics.com/tools/mount-disk-images.html) (virtual floppy image mount)
+
+**IMPORTANT: After installing this software, remember to append each program's executable path to Windows PATH variable.**
+
+## Get DreamOS
+
+To clone DreamOS repository, type:
+
+	cd $HOME
+    git clone https://github.com/inuyasha82/DreamOs
+    
+in cygwin terminal.
+
+## Edit .bochsrc
+
+Now if you have got the DreamOS folder, you will se a file named .bochsrc. Open it with an editor.
+
+Comment line 
+
+	display_library: x
+    
+with
+    
+    #display_library: x
+    
+Then add the line
+
+	display_library: win32
+    
+Supposing **C:\Program Files\Bochs-2.6.8\\** is your bochs installation path:   
+Substitute line
+
+	romimage: file=/home/vetinari/utils/bochs-2.6.7/bios/BIOS-bochs-latest
+    
+with
+
+	romimage: file="C:\Program Files\Bochs-2.6.8\BIOS-bochs-latest"
+    
+Substitute line
+
+	vgaromimage: file=/home/vetinari/utils/bochs-2.6.7/bios/VGABIOS-lgpl-latest
+    
+with
+
+	vgaromimage: file="C:\Program Files\Bochs-2.6.8\VGABIOS-lgpl-latest"
+    
+Substitute line
+
+	update_freq=10
+    
+with
+
+	update_freq=40
+    
+to have better video frequency performance
+
+Comment line
+
+	gdbstub: enabled=1, port=1234, text_base=0, data_base=0, bss_base=0
+    
+as
+
+	#gdbstub: enabled=1, port=1234, text_base=0, data_base=0, bss_base=0
+
+## Edit Makefile
+
+Remove lines
+
+	ifeq ($(shell which sudo),)

```

Comment:
Instead of removing lines, i should look on a conditional way to do it. Are we on windows? Then skip these lines, else...
Imagine a windows user that wants to contribute, he must have his own copy of the Makefile and everytime he has to make changes to it, he must re-add the lines above. (ok, this is pure sci-fi, i don't think that the os will face a huge crowd of contributors in the future :D )